pkg/controller/clusterinst_api.go

```diff
@@ -231,84 +221,63 @@ func (s *ClusterInstApi) CreateClusterInst(in *edgeproto.ClusterInst, cb edgepro
 	return s.createClusterInstInternal(DefCallContext(), in, cb)
 }
 
-type InfraResPrint struct {
-	edgeproto.InfraResource
-	Req uint64
-}
-
-// Validate resource requirements for the VMs on the cloudlet
-func (s *ClusterInstApi) validateCloudletInfraResources(ctx context.Context, stm concurrency.STM, cloudlet *edgeproto.Cloudlet, cloudletInfo *edgeproto.CloudletInfo, infraResources *edgeproto.InfraResourcesSnapshot, usedResources, reqdResources, subtractReqdResources *CloudletResources) ([]string, error) {
-	log.SpanLog(ctx, log.DebugLevelApi, "Validate cloudlet resources", "cloudlet", cloudlet.Key)
-
-	infraResInfo := make(map[string]edgeproto.InfraResource)
-	for _, resInfo := range infraResources.Info {
-		infraResInfo[resInfo.Name] = resInfo
-	}
-
-	reqdVals, err := s.all.cloudletApi.totalCloudletResources(ctx, stm, cloudlet, cloudletInfo, reqdResources, infraResInfo)
-	if err != nil {
+func (s *CloudletResCalc) cloudletFitsReqdVals(ctx context.Context, reqdVals resspec.ResValMap) ([]string, error) {
```

Comment:
You're right, I just thought it would be easier to review if I left it in place, then you can easily see what changed in this function. But looking at it now, there are so many small changes that the diff is not really that nice, so I'll just go ahead and move it.